text/lang-detection/Project.toml

```
@@ -3,11 +3,13 @@ Cascadia = "54eefc05-d75b-58de-a785-1a3403f0919f"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Gumbo = "708ec375-b3d6-5a57-a7ce-8257bf98657a"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
-Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
-Flux = "0.11.4"
-julia = "1.5"
+Cascadia = "1.0.2"
+Flux = "0.13.9"
+Gumbo = "0.8.2"
+HTTP = "1.7.4"
+julia = "1.6"
```

```suggestion
Cascadia = "1"
Flux = "0.13.9"
Gumbo = "0.8"
HTTP = "1.7"
julia = "1.6"

```
Just simplifying compat a bit and adding back in the final newline.